python/pyspark/ml/tests/test_algorithms.py

```diff
@@ -83,7 +83,7 @@ def test_raw_and_probability_prediction(self):
         result = model.transform(test).head()
         expected_prediction = 2.0
         expected_probability = [0.0, 0.0, 1.0]
-        expected_rawPrediction = [57.3955, -124.5462, 67.9943]
+        expected_rawPrediction = [-11.6081922998, -8.15827998691, 22.17757045]
```

Comment:
Although I'm surprised by the magnitude of this change, this result sounds more sensible. I don't think it's an issue.